### PR TITLE
Documentación modulo de telemedidas

### PR DESCRIPTION
**Pull request by MiquelIR**
### Description:

Adds a new page on the Distribuidora block for the Telemeasure module. 

Modifies the documentation for CCH in Comercialitzadora. Adding the docs for the F1 and P1 files.

### Tasks:

- [x] Docs for telemeasure
- [x] New files F1 and P1 imports for comer

### Links:

* [telemedida.md #Documentació del mòdul de telemesura](http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/ca/distri/telemedida/)
* [telemedida.md #Documentación del modulo de telemedida](http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/es/distri/telemedida/)

* [curvas.md #Funcionament #Configurar proveïdors #Nom dels fitxers de Corbes](http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/ca/comer/curvas/)
* [curvas.md #Funcionamiento #Confirmar proveedores #Nombre de los ficheros de Curvas](http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/es/comer/curvas/)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/telemedida.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# Documentació del mòdul de telemesura
+
+
+## Comptadors de telemesura i electrònics
+Aquest mòdul gestiona dos tipus de comptadors diferents, els comptadors
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Posa-ho en un parell de paràgrafs que no és gens amigable...

File: docs/distri/telemedida.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+## Configuració dels Comptadors
+Per tal de poder utilitzar aquest mòdul correctament s'hauran de configurar
+els comptadors que l'hagin de fer servir.
+
+### Configuració de la tecnologia
+Primer de tot s'haurà de configurar el comptador per tal que l'ERP el reconegui
+com a electrònic o electrònic tele-mesurat. Per fer-ho hem de modificar el camp
+**Tecnologia** de la fitxa del comptador i assignar-l'hi el valor **Electrònic**
+o **Telemesura**.
```
Comment (jaumef) [inline docs/distri/telemedida.md]: (ES) Telemdida → Telemedida

File: docs/distri/telemedida.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+decidir si són o no vàlids.
+
+1.  **Neteja de duplicats**: El primer pas de la validació busca tots els perfils que
+tinguin duplicats i els elimina deixant només un dels registres.
+
+2.  **Diferents mesures**: Es comprova que no existeixin registres per la mateixa hora
+i que tinguin valors de mesura diferents.
+
+3.  **Mesures impossibles**: Comprova que un perfil no tingui un valor de mesura massa
+elevat i que es consideri fora dels límits. Aquests límit es pot configurar a la variable
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Afegeix una nota (tip, note o info) per explicar que hi ha la variable de configuració

File: docs/distri/telemedida.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+**tm_profile_impossible**.
+
+4.  **Mesures negatives**: Assegura que no hi hagi cap perfil que contingui una valor de
+mesura negatiu.
+
+5.  **Bits de control**: Comprova que els registres no tinguin un valor al camp de bits
+de control que indiqui error.
+
+6.  **Reactiva alta**: Comprova que no hi hagi cap perfil que tingui una reactiva al
+quadrant 1 amb un valor superior a l'activa entrant. Aquesta validació es pot activar
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Afegeix una nota (tip, note o info) per explicar que hi ha la variable de configuració

File: docs/distri/telemedida.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+i desactivar amb la variable de configuració **tm_reactive_check**.
+
+7.  **Superior a la potència**: Comprova que cap perfil tingui un valor d'activa
+entrant superior al màxim permès segons la potència contractada al període en que es
+troba el perfil. Aquesta validació té un paràmetre de tolerància que serveix per
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Explica això del paràmetre i l'exemple amb una nota (tip, note o info), potser la variable amb un info i l'exemple amb un note, així quedarà separat i no serà tant carregós

File: docs/distri/telemedida.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+## Generació de fitxers de perfils F1
+Els fitxers F1 de perfils són diaris per tant cada un conté les 24 corbes horàries
+que corresponen. La generació d'aquests fitxers la porta a terme l'assistent
+**Infraestructura > Exportar REE F1**.
+
+![](_static/telemedida/ExportREEF1Menu.png)
+
+L'assistent necessita que li indiquem per quin comptador i quin període volem
+generar els F1s. Per posar un nom correcte als fitxer generats, és necessari indicar
+el codi R1 de la nostra distribuïdora. Si el tenim ben configurat al ERP l'assistent
```
Comment (jaumef) [inline docs/distri/telemedida.md]: "Si el tenim ben configurat al ERP l'assistent l'agafarà automàticament"

Posa-ho amb una nota i diguent com es fa

File: docs/distri/telemedida.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+l'agafarà automàticament. També disposem del camp **Versió** el qual ens servirà per
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Per cada camp que expliquis, si ho poses amb un nou paràgraf, no serà tant carregós

File: docs/distri/telemedida.md [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+enumerar els fitxers generats, és a dir, si generem més d'un F1 per el mateix comptador
+i dates, la versió ens servirà per diferenciar-los i saber quin és el més nou.
+Per crear els fitxers F1 de tot un mes sencer, podem posar com a data inicial el primer
```
Comment (jaumef) [inline docs/distri/telemedida.md]: Entenc que si fas de 1/mes a 1/mes seguent, no s'inclou l'última data, ja que deuen ser lectures fins a la data. Això no ho estàs explicant enlloc.
Si es només per un dia, has de fer data del dia i data del dia següent?

Per altra banda, pots posar tot l'exemple amb un "Tip", "note" o "info" incloent la imatge

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/telemedida.md [hunk 9]
```diff
@@ -111,34 +112,46 @@ iniciarà una validació per tots els comptadors de telemesura.
...
-tolerància s'utilitza la variable de configuració **tm_profile_power_tolerance**. Al mateix
-temps, si el valor d'aquesta variable es configura a 0, la validació es desactiva.
+del perfil.
+
+    !!! Info "Variable"
+        Existeix una variable de configuració anomenada **tm_profile_power_tolerance** que
+        ens permetrà configurar aquesta tolerància . Al mateix temps, si el valor d'aquesta
+        variable es configura a 0, la validació es desactiva.
+
+    !!! Note "Exemple"
```
Comment (jaumef) [inline docs/distri/telemedida.md]: The tranlation isn't right, replace 
`<p class=\"admonition-title\">Exemple</p>`
With 
`<p class=\"admonition-title\">Ejemplo</p>`


image

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_telemeasure_module_documentation/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**